app/views/user_sessions/_form.html.erb

```diff
@@ -80,14 +80,16 @@
       $('#loginModal').modal(); // Open the login modal to allow user to login
     });
 
+    var toggleState = false;
+
     $("#visibility").on("click", function(e) {
       e.preventDefault();
-      $(this).toggleClass("fa-eye-slash");
+      $(this).toggleClass("fa-eye fa-eye-slash").attr("title", toggleState ? "Show Password" : "Hide Password");
```

Comment:
Hi @divyabaid16 
Le't's just the use the tooltip used all over the plots2 rather than just setting the title.
What do you think ?